router/middleware.go

```diff
@@ -53,6 +54,22 @@ func getSessionFromCookieOrHeader(c *gin.Context) {
 	c.Next()
 }
 
+func _debugBypassEmptyLogin(c *gin.Context) {
```

Comment:
For functions typically wouldn’t use underscore.  Just lower case is sufficient if don’t want it exported.

Middleware for me usually ends up in a package and ends up needing to export the functions.  But for now I’d suggest removing _